packages/@aws-cdk/aws-codepipeline-actions/lib/codebuild/build-action.ts

```diff
@@ -102,6 +102,15 @@ export interface CodeBuildActionProps extends codepipeline.CommonAwsActionProps
    * @default false
    */
   readonly executeBatchBuild?: boolean;
+
+  /**
+   * Combine the build artifacts for a batch builds.
+   *
+   * Enabling this will enable to combine the build artifacts into the same location for a batch builds on the CodeBuild project.
```

Comment:
```suggestion
   * Enabling this will combine the build artifacts into the same location for batch builds.
   * If `executeBatchBuild` is not set to `true`, this property is ignored.
```